app/routes/application.js

```diff
@@ -102,7 +102,7 @@ export default Ember.Route.extend({
     if (error && _.isString(error)) {
       return error;
     }
-    return reason.errors[0].status == 403
+    return _.get(reason, "errors[0].status") == 403
```

Comment:
@patrixr why have we changed this ? I think the `reason` is passed to this function right ?

Reply:
This was breaking when reason didn't have an `error` property.
Using `_.get` does the same thing excepts it won't crash if a nested property doesn't exist

Reply:
It's the equivalent of doing `reason && reason.errors && reason.errors[0] && reasons.errors[0].status`